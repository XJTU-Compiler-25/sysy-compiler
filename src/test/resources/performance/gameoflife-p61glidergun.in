version https://git-lfs.github.com/spec/v1
oid sha256:977ab5a56b7fa2773760db07b1d86c73f7d439afe4394bcf35d40f0fad814f96
size 86937
