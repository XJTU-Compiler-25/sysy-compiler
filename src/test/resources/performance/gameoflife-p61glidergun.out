version https://git-lfs.github.com/spec/v1
oid sha256:f6805245a9a500ebd32b958b92e3e1636b1fb451dab8d52b263f47e82e886951
size 86622
